google/cloud/spanner/sql_statement.h

```diff
@@ -67,6 +67,13 @@ class SqlStatement {
   google::cloud::StatusOr<Value> GetParameter(
       std::string const& parameter_name) const;
 
+  friend bool operator==(SqlStatement const& a, SqlStatement const& b) {
+    return a.statement_ == b.statement_ && a.params_ == b.params_;
+  }
+  friend bool operator!=(SqlStatement const& a, SqlStatement const& b) {
+    return !(a == b);
+  };
```

Comment:
nit: extra `;` at the end of the function definition.

Reply:
Thanks! Removed